IdentityCore/src/cache/MSIDKeychainTokenCache.m

```diff
@@ -152,11 +152,7 @@ - (nullable instancetype)initWithGroup:(nullable NSString *)keychainGroup error:
                                                   (id)kSecAttrAccessGroup : self.keychainGroup} mutableCopy];
     
 #ifdef __MAC_OS_X_VERSION_MAX_ALLOWED
-#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 101500
```

Comment:
Minimum version for macOS is 10.15 (101500) so all checks like this have been removed as they are now always true